pkg/features/embedded/cloud-config-essentials/system/oem/00_rootfs.yaml

```diff
@@ -19,14 +19,14 @@ stages:
       environment:
         OVERLAY: "tmpfs:25%"
   initramfs:
-    - if: '[ ! -f "/run/cos/recovery_mode" ]'
+    - if: '[ ! -f "/run/elemental/recovery_mode" ]'
       name: "Persist /etc/machine-id"
       commands:
       - |
         # persist machine-id
-        if [ -s /usr/local/etc/machine-id ]; then
-          cat /usr/local/etc/machine-id > /etc/machine-id
+        if [ -s /run/elemental/persistent/etc/machine-id ]; then
+          cat /run/elemental/persistent/etc/machine-id > /etc/machine-id
```

Comment:
I assume this is because the persistent partition is no longer mounted at `/usr/local` by default, right?

Reply:
Yes, that is correct